mesonbuild/compilers/compilers.py

```diff
@@ -325,7 +327,7 @@ class CompileCheckMode(enum.Enum):
     OptionKey('b_pie'): coredata.UserBooleanOption('Build executables as position independent', False),
     OptionKey('b_bitcode'): coredata.UserBooleanOption('Generate and embed bitcode (only macOS/iOS/tvOS)', False),
     OptionKey('b_vscrt'): coredata.UserComboOption('VS run-time library type to use.',
-                                                   ['none', 'md', 'mdd', 'mt', 'mtd', 'from_buildtype', 'static_from_buildtype'],
+                                                   list(MSCRT_VALS) + ['from_buildtype', 'static_from_buildtype'],
```

Comment:
You either need `sorted(MSVCRT_VALS)` here, or to use an `OrderedSet` for this, otherwise the `meson configure` output may be re-ordered

Reply:
Good catch. Made it a list, for small set it does not really make a difference.